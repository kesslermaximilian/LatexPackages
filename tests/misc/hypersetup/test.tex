\documentclass{article}

\usepackage[cleveref]{mkessler-hypersetup}

\title{Geometrie}
\author{Maximilian Ke√üler}

\begin{document}
    \maketitle
    test

    \section{test}
    \label{foo}

    \Cref{foo} test.
\end{document}
