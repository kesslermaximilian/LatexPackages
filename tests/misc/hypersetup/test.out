\BOOKMARK [1][-]{section.1}{test}{}% 1
